text-ui-test/ACTUAL.TXT

```diff
@@ -0,0 +1,31 @@
+    ____________________________________________________________
```

Comment:
maybe this does not need to be committed since this is generated new every time the code is run and depends on the program